app/controllers/employee_positions_controller.rb

```diff
@@ -0,0 +1,62 @@
+class EmployeePositionsController < ApplicationController
+  include Draper::Decoratable
+  before_action :set_employee, only: [:create, :new, :update]
+  before_action :set_employee_position, only: [:update]
+
+  def index
+    @employee = Employee.find(params[:id])
```

Comment:
you already have `set_employee` method for this logic.

Reply:
It differs. I have employee id and employee position id.